src/py21cmfast/_cfg.py

```diff
@@ -39,7 +39,10 @@ def __init__(self, *args, write=True, file_name=None, **kwargs):
         for k, v in self._defaults.items():
             if k not in self:
                 if k not in self._aliases:
-                    warnings.warn("Your configuration file is out of date. Updating...")
+                    if self.file_name:
```

Comment:
If I look at v4-prep in the normal code viewer these changes are already there. It must be comparing to an older version before I merged in master

Reply:
Hmmmm weird -- it should have updated the thing it's pointing at